Source/src/modules/ModuleEditor.h

```diff
@@ -54,7 +54,10 @@ namespace Hachiko
 
         void SetSelectedGO(GameObject* const selected_game_object)
         {
-            this->selected_go = selected_game_object;
+            if (selected_game_object != selected_go)
```

Comment:
```suggestion
            selected_go = selected_game_object;
```
Hey Bernat! Thanks for the good work. However, testing your code, if you try selecting `GameObject`s on `Scene` directly with mouse raycasting (not from the hierarchy). This piece of code couses glitchy toggling at the inspector between `nullptr` and hit `GameObject` by the raycasting. I can see this working for selecting `GameObject`s from the hierarchy so that if user selects the same `GameObject` again inspector just renders blank and `selected_go` is set to `nullptr`. A quick solution would be reverting this back to previous version and handling this toggle stuff before calling this function from Hierarchy part, not inside this method.

As a side note, even though the `if` `else` blocks are single lines, our styling mandates that you use newlined braces.